tests/integration-tests/tests/iam/test_iam.py

```diff
@@ -145,9 +145,16 @@ def test_s3_read_write_resource(
     cluster_config = pcluster_config_reader(bucket=bucket_name)
     cluster = clusters_factory(cluster_config)
 
-    config = configparser.ConfigParser()
-    config.read(cluster_config)
+    with open(cluster_config) as conf_file:
+        config = yaml.load(conf_file, Loader=yaml.SafeLoader)
 
     # Check S3 resources
-    check_s3_read_resource(region, cluster, config.get("cluster default", "s3_read_resource"))
-    check_s3_read_write_resource(region, cluster, config.get("cluster default", "s3_read_write_resource"))
+    check_s3_read_resource(region, cluster, _get_bucket_name(config, False))
```

Comment:
nit: let's use keyword when passing a boolean parameter, in this way it's easier to understand what we're passing from the caller. `_get_bucket_name(config, enable_write_access=True|False)`

Reply:
Done